src/components/ServiceAreasDrawer/ServiceAreasDrawer.css

```diff
@@ -0,0 +1,18 @@
+@import "../../index.css";
+
+/* Don't cover up Download CSV link */
+.LeftDrawer .ZipCodeSelector {
+  overflow-y: auto;
+  position: absolute;
+    bottom: 124px;
+    right: 20px;
+    left: 20px;
+    top: 275px;
+}
+
+.LeftDrawer .ClearInputsButton {
```

Comment:
@bcherny  ended up setting this here, but then it impacts on both panels. Should I be setting this in LeftPane.css? or in ClearInputsButton.css instead?

Reply:
Not sure I understand and I'm not at a computer to repro - do you think you could elaborate?

Reply:
This is affecting each time there's a ClearInputsButton in a Left Drawer, even when it's not in service area. I'm going to go ahead and move it to LeftPane.css